Web/View/Layout.hs

```diff
@@ -44,13 +44,11 @@ scripts = [hsx|
         <script src={assetPath "/vendor/jquery-3.6.0.slim.min.js"}></script>
         <script src={assetPath "/vendor/timeago.js"}></script>
         <script src={assetPath "/vendor/flatpickr.js"}></script>
-        <script src={assetPath "/vendor/morphdom-umd.min.js"}></script>
-        <script src={assetPath "/vendor/turbolinks.js"}></script>
-        <script src={assetPath "/vendor/turbolinksInstantClick.js"}></script>
-        <script src={assetPath "/vendor/turbolinksMorphdom.js"}></script>
```

Comment:
The version of Turbolinks here is adapted to IHP, so it should be working fine.

Reply:
I tried keeping Turbolinks but it seems its cache system breaks tinymce initialization (spohlenz/tinymce-rails#188), after looking for a while I found a way to disable turbolinks for specific pages (https://github.com/turbolinks/turbolinks#disabling-turbolinks-on-specific-links). It would work, but it will demand that we build a way to detect when tinymce is being used in a target URL. As turbolink is [unmaintained](https://github.com/turbolinks/turbolinks#turbolinks-is-no-longer-under-active-development), I believe it does not make sense to open a issue there.

Another alternative is to disable turbolinks manually for each page using tinymce.

Reply:
Which versions you're using there? Let me try here